### PR TITLE
Log SQL with values for insert/update statements

### PR DESCRIPTION
**Pull request by chipz**
Hi!

I want to try to close issue #1317, at first I saw pull request #1810 that will also close this issue, but for a long time the author doesn't reply to @d6y [comment](https://github.com/slick/slick/pull/1810#issuecomment-341989355). 

I [asked](https://github.com/slick/slick/pull/1810#issuecomment-401968551) if I can create a new PR with the changes asked by @d6y, after I get the go, I created this pull request.

So I created an additional logger `JdbcBackend.statementAndParameters` because we want to avoid changing the current behaviour of `JdbcBackend.statement`. Just like the original PR author @landlockedsurfer said. Adding the logger:

```
<logger name="slick.jdbc.JdbcBackend.statementAndParameter" level="DEBUG" />
```

will log executable SQL statements with real value instead of `?`

Cheers,
Fajr

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @chipz,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by chipz** [comment]
I already did sign the CLA

**Comment by d6y** [comment]
Thank you @chipz  - this works for me 🎉 

I've tried it on a small "hello-world-ish" project and I see the values logged for insert (`+=`) and for a select as expected. For example:

```
DEBUG s.j.J.statementAndParameter - Executing prepared update: prep1: insert into "message" ("sender","content")  values (?,?) {1: 'Dave', 2: 'Hello, HAL. Do you read me, HAL?'}
```

and

```
DEBUG s.j.J.statementAndParameter - Executing prepared statement: prep2: select "sender", "content", "id" from "message" where "id" = 1
```

I don't see output for batch insert (`++=`) but that's fine with me. Could always be a future thing, if it's possible.

TL;DR; :+1:
